### PR TITLE
Postopedia private wikis

### PR DESCRIPTION
**Pull request by mar1965**
Hi Britt,

So much for taking advantage of your time off yesterday ;) but am gonna sound like a broken record and will spare any details. Hope the root canal work went smoothly!!

Anyhow, somehow, something broke during my last push the other day, and I probably missed it. I just pushed some minor changes to the Downgrade, but am so confused right now because the behavior is not what I am expecting even when upgrading, something that we tested and had been proven to work.

Basically, after an upgrade, some wikis disappear from the view of the premium user. And then now, a downgraded user can see the Private checkbox and actually edit the wiki and make it public. Am wondering if something broke the other day and we missed it during our session.

Grrrr, my commit message is obviously wrong since the Downgrade isn't working yet.

Thank you!

### CODE REVIEW

**Comment by wonderwoman13** [comment]
Hi @mar1965 - I will happily look at this when I get home tonight. Thanks for pushing it up to Github!

**Comment by wonderwoman13** [comment]
@mar1965 the good news is that it was minor, you just forgot to set the premium user's wikis to public when they downgraded (essentially orphaned the wikis so that no-one could see them). I added in the logic and a test. Please review the commit. 

https://github.com/mar1965/postopedia/pull/6/commits/404e3d2bcb64f8d46ed43f69ad3b1099428367fe

Merging!

**Comment by mar1965** [comment]
Oh, that sounds great! Thank you! I will review the commit.
